README.md

```diff
@@ -166,9 +169,10 @@ Since `ForagePANEditText` is currently not receiving any style from your theme,
 
 <img src="screenshots/forage_pan_edit_text_no_style.png" width="300" height="500">
```

Comment:
Need to replace this image with an updated one

Reply:
Actually these images may not be stale and this can come later so I'm not gonna do this right now